docs/UserGuide.adoc

```diff
@@ -360,6 +366,64 @@ Opens menu to view, add, edit and remove Semester and Module data.
 
 Format: `semester`
 
+=== Task Tracker Commands
+A successful you begins with self-awareness!
+
+OOF, your Outstanding Organisation Friend, provides you with a simple and efficient way to keep track of the amount of time you spend on each module!
+
+==== Start tracker: `start`
+Begin your journey to optimum productivity by starting your task timer!
+Start your task tracking from current time.
+
+Format: `start MODULE_CODE`
+
+Example: `start CS2113T`
+
+image::demo_images/Demo_StartTracker.png[]
+
+==== Pause tracker: `pause`
+Not completed with your Task but want to take a break or get started with another task?
+Pause your task tracking at current time.
+
+Format: `pause MODULE_CODE`
+
+Example: `pause CS2113T`
+
+image::demo_images/Demo_PauseTracker.png[]
+
+==== Stop tracker: `stop`
+Complete with your Task? Stop task tracking at current time with `stop`.
+
+[NOTE]
+
+Your Task automatically will be marked `Done`.
+
+Format: `stop MODULE_CODE`
+
+Example: `stop CS2113T Activity Diagram`
+
+image::demo_images/Demo_StopTracker.png[]
+
+==== View tracker: `viewTracker`
+Want to view a visual display featuring the amount of time you spend on each of your `Module` over time?
+
+Format: `viewTracker`
+
+Example: `viewTracker`
+
+image::demo_images/Demo_ViewTracker.png[]
+
+Format: `viewTracker TIME_PERIOD [_coming soon in v1.4_]`
+
+*Options for TIME_PERIOD* +
+[horizontal]
+Day:: filter time spent on each `Module` today
+Week:: filter time spent on each `Module` over the course of the last 7 days
+
+Example: `viewTracker Day`
+
+image::demo_images/Demo_ViewTrackerTimePeriod.png[]
+
```

Comment:
Why is the task tracker tracking module when it is a TASK tracker and module is not a TASK. This feature is meant to track individual tasks such that the user can know how much time they spent on each task. Tracking time spent on a module does not provide any insightful details to the user. There is a reason why each module contains a list of assignments. "Time spent" belongs to the task (assignment) and when you navigate to a module, you can navigate to each individual assignment and view the time taken for that assignment.

Also, there should not be 4 different commands for the same functionality as it will cause confusion for the user and it does not fit with the command style that we use.

Lastly, the UG is missing the source files for your images. Is it intentional for the UG to be merged while incomplete?

Reply:
I have changed the implementation of the Tracker feature as suggested. UG has also been corrected.